Source/ASCollectionView.mm

```diff
@@ -960,30 +960,38 @@ - (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollection
 {
   ASDisplayNodeAssertMainThread();
   NSIndexPath *indexPath = [NSIndexPath indexPathForItem:0 inSection:section];
-  ASCellNode *cell = [self supplementaryNodeForElementKind:UICollectionElementKindSectionHeader
-                                               atIndexPath:indexPath];
-  if (cell.shouldUseUIKitCell && _asyncDelegateFlags.interop) {
+  ASCollectionElement *element = [_dataController.visibleMap supplementaryElementOfKind:UICollectionElementKindSectionHeader
+                                                                            atIndexPath:indexPath];
+  if (element == nil) {
+    return CGSizeZero;
```

Comment:
@appleguy In case of interop, should we reach out to the delegate here? Note that since element is `nil`, `element.node.shouldUseUIKitCell` must be `NO` here, so this is not a breaking change.